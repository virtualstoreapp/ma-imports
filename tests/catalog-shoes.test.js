/**
 * @jest-environment jest-environment-jsdom
 */
const fs = require('fs');
const path = require('path');
const { selectCategory, selectMobileCategory } = require('./catalogTestUtils');
require('@testing-library/jest-dom');
const { setupGlobalFetchMock } = require('./catalogTestUtils');

global.gtag = jest.fn();

beforeAll(() => {
  setupGlobalFetchMock();
});

describe('Catalog Page - Shoes', () => {
  beforeEach(() => {
    window.location.hash = '';
    const html = fs.readFileSync(path.resolve(__dirname, '../index.html'), 'utf8');
    document.documentElement.innerHTML = html;
    setupDOM();
  });

  describe('Desktop View', () => {
    it('Matches snapshot for "Shoes"', async () => {
      await selectCategory("shoes", "Calçados");
      expect(document.body.innerHTML).toMatchSnapshot();
    });
  });

  describe('Mobile View', () => {
    beforeEach(() => {
      window.innerWidth = 375;
      window.dispatchEvent(new Event('resize'));
      expect(document.getElementById('menu-toggle')).toBeInTheDocument();
    });

    it('Matches snapshot for "Shoes" on mobile', async () => {
      await selectMobileCategory("shoes", "Calçados");
      expect(document.body.innerHTML).toMatchSnapshot();
    });
  });
});

// Helper to set up the DOM for each test
function setupDOM() {
  jest.resetModules();
  require('../scripts.js');
  document.dispatchEvent(new Event('DOMContentLoaded'));
}
