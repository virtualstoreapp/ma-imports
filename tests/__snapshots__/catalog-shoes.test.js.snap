// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Catalog Page - Shoes Desktop View Matches snapshot for "Shoes" 1`] = `
"
  <header>
    <div class="header-logo">
      <img src="images/logo.jpeg" alt="M/A Imports Logo">
    </div>
  </header>

  <nav aria-label="Main Navigation">
    <!-- Mobile Menu Toggle Button -->
    <button id="menu-toggle" aria-expanded="false" aria-controls="nav-menu">☰</button>
    <ul id="nav-menu">
      <li><button data-category="all">Início</button></li>
      <li><button data-category="shoes">Calçados</button></li>
      <li><button data-category="slippers">Chinelos</button></li>
    </ul>
  </nav>

  <main>
    <!-- Category heading (updates based on selection) -->
    <h2 id="category-heading">Calçados</h2>
    <!-- All products will be rendered inside this container -->
    <ul id="product-list"></ul>
  </main>

  <!-- Footer with visible app version -->
  <footer>
    <p>Versão: 1.0.4 - 0.28</p>
  </footer>

  <script src="scripts.js"></script>


"
`;

exports[`Catalog Page - Shoes Mobile View Matches snapshot for "Shoes" on mobile 1`] = `
"
  <header>
    <div class="header-logo">
      <img src="images/logo.jpeg" alt="M/A Imports Logo">
    </div>
  </header>

  <nav aria-label="Main Navigation" class="">
    <!-- Mobile Menu Toggle Button -->
    <button id="menu-toggle" aria-expanded="false" aria-controls="nav-menu">☰</button>
    <ul id="nav-menu">
      <li><button data-category="all">Início</button></li>
      <li><button data-category="shoes">Calçados</button></li>
      <li><button data-category="slippers">Chinelos</button></li>
    </ul>
  </nav>

  <main>
    <!-- Category heading (updates based on selection) -->
    <h2 id="category-heading">Calçados</h2>
    <!-- All products will be rendered inside this container -->
    <ul id="product-list"><li class="product-item">
        <img src="images/shoes/nike/12021658-nike-shox.jpeg" alt="[12021658] Nike Shox">
        <div class="product-details">
          <h3>[12021658] Nike Shox</h3>
          <span class="size">Tamanho: 37</span>
          <span class="old-price">R$&nbsp;259,00</span>
           <span class="new-price">R$&nbsp;249,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/nike/12022216-nike-shox.jpeg" alt="[12022216] Nike Shox">
        <div class="product-details">
          <h3>[12022216] Nike Shox</h3>
          <span class="size">Tamanho: 37</span>
          <span class="old-price">R$&nbsp;259,00</span>
           <span class="new-price">R$&nbsp;249,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/nike/12021724-nike-shox.jpeg" alt="[12021724] Nike Shox">
        <div class="product-details">
          <h3>[12021724] Nike Shox</h3>
          <span class="size">Tamanho: 40</span>
          <span class="old-price">R$&nbsp;269,00</span>
           <span class="new-price">R$&nbsp;259,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/nike/12022214-nike-shox.jpeg" alt="[12022214] Nike Shox">
        <div class="product-details">
          <h3>[12022214] Nike Shox</h3>
          <span class="size">Tamanho: 40</span>
          <span class="old-price">R$&nbsp;269,00</span>
           <span class="new-price">R$&nbsp;259,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/nike/12021722-nike-shox-neymar.jpeg" alt="[12021722] Nike Shox Neymar">
        <div class="product-details">
          <h3>[12021722] Nike Shox Neymar</h3>
          <span class="size">Tamanho: 41</span>
          <span class="old-price">R$&nbsp;269,00</span>
           <span class="new-price">R$&nbsp;259,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/nike/12021734-nike-shox-neymar.jpeg" alt="[12021734] Nike Shox Neymar">
        <div class="product-details">
          <h3>[12021734] Nike Shox Neymar</h3>
          <span class="size">Tamanho: 41</span>
          <span class="old-price">R$&nbsp;269,00</span>
           <span class="new-price">R$&nbsp;259,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/nike/12021718-nike-r4.jpeg" alt="[12021718] Nike R4">
        <div class="product-details">
          <h3>[12021718] Nike R4</h3>
          <span class="size">Tamanho: 34</span>
          <span class="old-price">R$&nbsp;229,00</span>
           <span class="new-price">R$&nbsp;219,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/nike/12021716-nike-r4.jpeg" alt="[12021716] Nike R4">
        <div class="product-details">
          <h3>[12021716] Nike R4</h3>
          <span class="size">Tamanho: 35</span>
          <span class="old-price">R$&nbsp;229,00</span>
           <span class="new-price">R$&nbsp;219,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/nike/12021703-nike-air-force.jpeg" alt="[12021703] Nike Air Force">
        <div class="product-details">
          <h3>[12021703] Nike Air Force</h3>
          <span class="size">Tamanho: 39</span>
          <span class="old-price">R$&nbsp;219,00</span>
           <span class="new-price">R$&nbsp;209,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/nike/12021710-nike-air-force.jpeg" alt="[12021710] Nike Air Force">
        <div class="product-details">
          <h3>[12021710] Nike Air Force</h3>
          <span class="size">Tamanho: 42</span>
          <span class="old-price">R$&nbsp;269,00</span>
           <span class="new-price">R$&nbsp;259,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/nike/12021739-nike-air-force.jpeg" alt="[12021739] Nike Air Force">
        <div class="product-details">
          <h3>[12021739] Nike Air Force</h3>
          <span class="size">Tamanho: 42</span>
          <span class="old-price">R$&nbsp;219,00</span>
           <span class="new-price">R$&nbsp;209,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/nike/12021747-nike-air-force.jpeg" alt="[12021747] Nike Air Force">
        <div class="product-details">
          <h3>[12021747] Nike Air Force</h3>
          <span class="size">Tamanho: 38, 41, 42</span>
          <span class="old-price">R$&nbsp;229,00</span>
           <span class="new-price">R$&nbsp;219,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/nike/12021737-nike-dunk-low-pro.jpeg" alt="[12021737] Nike Dunk Low Pro">
        <div class="product-details">
          <h3>[12021737] Nike Dunk Low Pro</h3>
          <span class="size">Tamanho: 35</span>
          <span class="old-price">R$&nbsp;159,00</span>
           <span class="new-price">R$&nbsp;149,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/nike/12021719-nike-dunk-low-pro.jpeg" alt="[12021719] Nike Dunk Low Pro">
        <div class="product-details">
          <h3>[12021719] Nike Dunk Low Pro</h3>
          <span class="size">Tamanho: 36</span>
          <span class="old-price">R$&nbsp;159,00</span>
           <span class="new-price">R$&nbsp;149,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/nike/12022152-nike-dunk-low-pro.jpeg" alt="[12022152] Nike Dunk Low Pro">
        <div class="product-details">
          <h3>[12022152] Nike Dunk Low Pro</h3>
          <span class="size">Tamanho: 39, 42</span>
          <span class="old-price">R$&nbsp;219,00</span>
           <span class="new-price">R$&nbsp;209,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/nike/12021733-nike-dunk-low-pro.jpeg" alt="[12021733] Nike Dunk Low Pro">
        <div class="product-details">
          <h3>[12021733] Nike Dunk Low Pro</h3>
          <span class="size">Tamanho: 42</span>
          <span class="old-price">R$&nbsp;219,00</span>
           <span class="new-price">R$&nbsp;209,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/nike/12021707-nike-jordan.jpeg" alt="[12021707] Nike Jordan">
        <div class="product-details">
          <h3>[12021707] Nike Jordan</h3>
          <span class="size">Tamanho: 38</span>
          <span class="old-price">R$&nbsp;169,00</span>
           <span class="new-price">R$&nbsp;159,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/nike/12022200-nike-air-jordan.jpeg" alt="[12022200] Nike Air Jordan">
        <div class="product-details">
          <h3>[12022200] Nike Air Jordan</h3>
          <span class="size">Tamanho: 41</span>
          <span class="old-price">R$&nbsp;229,00</span>
           <span class="new-price">R$&nbsp;219,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/adidas/12022201-adidas-campus.jpeg" alt="[12022201] Adidas Campus">
        <div class="product-details">
          <h3>[12022201] Adidas Campus</h3>
          <span class="size">Tamanho: 35</span>
          <span class="old-price">R$&nbsp;159,00</span>
           <span class="new-price">R$&nbsp;149,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/adidas/12022202-adidas-campus.jpeg" alt="[12022202] Adidas Campus">
        <div class="product-details">
          <h3>[12022202] Adidas Campus</h3>
          <span class="size">Tamanho: 35</span>
          <span class="old-price">R$&nbsp;259,00</span>
           <span class="new-price">R$&nbsp;249,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/adidas/12022208-adidas-campus.jpeg" alt="[12022208] Adidas Campus">
        <div class="product-details">
          <h3>[12022208] Adidas Campus</h3>
          <span class="size">Tamanho: 36</span>
          <span class="old-price">R$&nbsp;159,00</span>
           <span class="new-price">R$&nbsp;149,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/adidas/12022211-adidas-campus.jpeg" alt="[12022211] Adidas Campus">
        <div class="product-details">
          <h3>[12022211] Adidas Campus</h3>
          <span class="size">Tamanho: 39</span>
          <span class="old-price">R$&nbsp;159,00</span>
           <span class="new-price">R$&nbsp;149,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/adidas/12022155-adidas-campus.jpeg" alt="[12022155] Adidas Campus">
        <div class="product-details">
          <h3>[12022155] Adidas Campus</h3>
          <span class="size">Tamanho: 41</span>
          <span class="old-price">R$&nbsp;159,00</span>
           <span class="new-price">R$&nbsp;149,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/adidas/12022210-adidas-campus.jpeg" alt="[12022210] Adidas Campus">
        <div class="product-details">
          <h3>[12022210] Adidas Campus</h3>
          <span class="size">Tamanho: 42</span>
          <span class="old-price">R$&nbsp;159,00</span>
           <span class="new-price">R$&nbsp;149,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/vans/12021757-vans.jpeg" alt="[12021757] Vans">
        <div class="product-details">
          <h3>[12021757] Vans</h3>
          <span class="size">Tamanho: 36</span>
          <span class="old-price">R$&nbsp;259,00</span>
           <span class="new-price">R$&nbsp;249,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/asics/12021714-asics.jpeg" alt="[12021714] Asics">
        <div class="product-details">
          <h3>[12021714] Asics</h3>
          <span class="size">Tamanho: 38</span>
          <span class="price">R$&nbsp;89,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/asics/12021736-asics.jpeg" alt="[12021736] Asics">
        <div class="product-details">
          <h3>[12021736] Asics</h3>
          <span class="size">Tamanho: 41, 42</span>
          <span class="old-price">R$&nbsp;269,00</span>
           <span class="new-price">R$&nbsp;259,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/nike/12021658-nike-shox.jpeg" alt="[12021658] Nike Shox">
        <div class="product-details">
          <h3>[12021658] Nike Shox</h3>
          <span class="size">Tamanho: 37</span>
          <span class="old-price">R$&nbsp;259,00</span>
           <span class="new-price">R$&nbsp;249,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/nike/12022216-nike-shox.jpeg" alt="[12022216] Nike Shox">
        <div class="product-details">
          <h3>[12022216] Nike Shox</h3>
          <span class="size">Tamanho: 37</span>
          <span class="old-price">R$&nbsp;259,00</span>
           <span class="new-price">R$&nbsp;249,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/nike/12021724-nike-shox.jpeg" alt="[12021724] Nike Shox">
        <div class="product-details">
          <h3>[12021724] Nike Shox</h3>
          <span class="size">Tamanho: 40</span>
          <span class="old-price">R$&nbsp;269,00</span>
           <span class="new-price">R$&nbsp;259,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/nike/12022214-nike-shox.jpeg" alt="[12022214] Nike Shox">
        <div class="product-details">
          <h3>[12022214] Nike Shox</h3>
          <span class="size">Tamanho: 40</span>
          <span class="old-price">R$&nbsp;269,00</span>
           <span class="new-price">R$&nbsp;259,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/nike/12021722-nike-shox-neymar.jpeg" alt="[12021722] Nike Shox Neymar">
        <div class="product-details">
          <h3>[12021722] Nike Shox Neymar</h3>
          <span class="size">Tamanho: 41</span>
          <span class="old-price">R$&nbsp;269,00</span>
           <span class="new-price">R$&nbsp;259,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/nike/12021734-nike-shox-neymar.jpeg" alt="[12021734] Nike Shox Neymar">
        <div class="product-details">
          <h3>[12021734] Nike Shox Neymar</h3>
          <span class="size">Tamanho: 41</span>
          <span class="old-price">R$&nbsp;269,00</span>
           <span class="new-price">R$&nbsp;259,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/nike/12021718-nike-r4.jpeg" alt="[12021718] Nike R4">
        <div class="product-details">
          <h3>[12021718] Nike R4</h3>
          <span class="size">Tamanho: 34</span>
          <span class="old-price">R$&nbsp;229,00</span>
           <span class="new-price">R$&nbsp;219,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/nike/12021716-nike-r4.jpeg" alt="[12021716] Nike R4">
        <div class="product-details">
          <h3>[12021716] Nike R4</h3>
          <span class="size">Tamanho: 35</span>
          <span class="old-price">R$&nbsp;229,00</span>
           <span class="new-price">R$&nbsp;219,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/nike/12021703-nike-air-force.jpeg" alt="[12021703] Nike Air Force">
        <div class="product-details">
          <h3>[12021703] Nike Air Force</h3>
          <span class="size">Tamanho: 39</span>
          <span class="old-price">R$&nbsp;219,00</span>
           <span class="new-price">R$&nbsp;209,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/nike/12021710-nike-air-force.jpeg" alt="[12021710] Nike Air Force">
        <div class="product-details">
          <h3>[12021710] Nike Air Force</h3>
          <span class="size">Tamanho: 42</span>
          <span class="old-price">R$&nbsp;269,00</span>
           <span class="new-price">R$&nbsp;259,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/nike/12021739-nike-air-force.jpeg" alt="[12021739] Nike Air Force">
        <div class="product-details">
          <h3>[12021739] Nike Air Force</h3>
          <span class="size">Tamanho: 42</span>
          <span class="old-price">R$&nbsp;219,00</span>
           <span class="new-price">R$&nbsp;209,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/nike/12021747-nike-air-force.jpeg" alt="[12021747] Nike Air Force">
        <div class="product-details">
          <h3>[12021747] Nike Air Force</h3>
          <span class="size">Tamanho: 38, 41, 42</span>
          <span class="old-price">R$&nbsp;229,00</span>
           <span class="new-price">R$&nbsp;219,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/nike/12021737-nike-dunk-low-pro.jpeg" alt="[12021737] Nike Dunk Low Pro">
        <div class="product-details">
          <h3>[12021737] Nike Dunk Low Pro</h3>
          <span class="size">Tamanho: 35</span>
          <span class="old-price">R$&nbsp;159,00</span>
           <span class="new-price">R$&nbsp;149,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/nike/12021719-nike-dunk-low-pro.jpeg" alt="[12021719] Nike Dunk Low Pro">
        <div class="product-details">
          <h3>[12021719] Nike Dunk Low Pro</h3>
          <span class="size">Tamanho: 36</span>
          <span class="old-price">R$&nbsp;159,00</span>
           <span class="new-price">R$&nbsp;149,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/nike/12022152-nike-dunk-low-pro.jpeg" alt="[12022152] Nike Dunk Low Pro">
        <div class="product-details">
          <h3>[12022152] Nike Dunk Low Pro</h3>
          <span class="size">Tamanho: 39, 42</span>
          <span class="old-price">R$&nbsp;219,00</span>
           <span class="new-price">R$&nbsp;209,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/nike/12021733-nike-dunk-low-pro.jpeg" alt="[12021733] Nike Dunk Low Pro">
        <div class="product-details">
          <h3>[12021733] Nike Dunk Low Pro</h3>
          <span class="size">Tamanho: 42</span>
          <span class="old-price">R$&nbsp;219,00</span>
           <span class="new-price">R$&nbsp;209,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/nike/12021707-nike-jordan.jpeg" alt="[12021707] Nike Jordan">
        <div class="product-details">
          <h3>[12021707] Nike Jordan</h3>
          <span class="size">Tamanho: 38</span>
          <span class="old-price">R$&nbsp;169,00</span>
           <span class="new-price">R$&nbsp;159,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/nike/12022200-nike-air-jordan.jpeg" alt="[12022200] Nike Air Jordan">
        <div class="product-details">
          <h3>[12022200] Nike Air Jordan</h3>
          <span class="size">Tamanho: 41</span>
          <span class="old-price">R$&nbsp;229,00</span>
           <span class="new-price">R$&nbsp;219,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/adidas/12022201-adidas-campus.jpeg" alt="[12022201] Adidas Campus">
        <div class="product-details">
          <h3>[12022201] Adidas Campus</h3>
          <span class="size">Tamanho: 35</span>
          <span class="old-price">R$&nbsp;159,00</span>
           <span class="new-price">R$&nbsp;149,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/adidas/12022202-adidas-campus.jpeg" alt="[12022202] Adidas Campus">
        <div class="product-details">
          <h3>[12022202] Adidas Campus</h3>
          <span class="size">Tamanho: 35</span>
          <span class="old-price">R$&nbsp;259,00</span>
           <span class="new-price">R$&nbsp;249,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/adidas/12022208-adidas-campus.jpeg" alt="[12022208] Adidas Campus">
        <div class="product-details">
          <h3>[12022208] Adidas Campus</h3>
          <span class="size">Tamanho: 36</span>
          <span class="old-price">R$&nbsp;159,00</span>
           <span class="new-price">R$&nbsp;149,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/adidas/12022211-adidas-campus.jpeg" alt="[12022211] Adidas Campus">
        <div class="product-details">
          <h3>[12022211] Adidas Campus</h3>
          <span class="size">Tamanho: 39</span>
          <span class="old-price">R$&nbsp;159,00</span>
           <span class="new-price">R$&nbsp;149,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/adidas/12022155-adidas-campus.jpeg" alt="[12022155] Adidas Campus">
        <div class="product-details">
          <h3>[12022155] Adidas Campus</h3>
          <span class="size">Tamanho: 41</span>
          <span class="old-price">R$&nbsp;159,00</span>
           <span class="new-price">R$&nbsp;149,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/adidas/12022210-adidas-campus.jpeg" alt="[12022210] Adidas Campus">
        <div class="product-details">
          <h3>[12022210] Adidas Campus</h3>
          <span class="size">Tamanho: 42</span>
          <span class="old-price">R$&nbsp;159,00</span>
           <span class="new-price">R$&nbsp;149,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/vans/12021757-vans.jpeg" alt="[12021757] Vans">
        <div class="product-details">
          <h3>[12021757] Vans</h3>
          <span class="size">Tamanho: 36</span>
          <span class="old-price">R$&nbsp;259,00</span>
           <span class="new-price">R$&nbsp;249,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/asics/12021714-asics.jpeg" alt="[12021714] Asics">
        <div class="product-details">
          <h3>[12021714] Asics</h3>
          <span class="size">Tamanho: 38</span>
          <span class="price">R$&nbsp;89,00</span>
        </div>
      </li><li class="product-item">
        <img src="images/shoes/asics/12021736-asics.jpeg" alt="[12021736] Asics">
        <div class="product-details">
          <h3>[12021736] Asics</h3>
          <span class="size">Tamanho: 41, 42</span>
          <span class="old-price">R$&nbsp;269,00</span>
           <span class="new-price">R$&nbsp;259,00</span>
        </div>
      </li></ul>
  </main>

  <!-- Footer with visible app version -->
  <footer>
    <p>Versão: 1.0.4 - 0.28</p>
  </footer>

  <script src="scripts.js"></script>


"
`;
